rpcs3/Emu/System.cpp

```diff
@@ -1709,12 +1709,23 @@ game_boot_result Emulator::Load(const std::string& title_id, bool is_disc_patch)
 		// Check game updates
 		const std::string hdd0_boot = hdd0_game + m_title_id + "/USRDIR/EBOOT.BIN";
 
-		if (!m_ar && disc.empty() && !bdvd_dir.empty() && !m_title_id.empty() && resolved_path == GetCallbacks().resolve_path(vfs::get("/dev_bdvd/PS3_GAME/USRDIR/EBOOT.BIN")) && resolved_path != GetCallbacks().resolve_path(hdd0_boot) && fs::is_file(hdd0_boot))
+		if (!m_ar && recursion_count == 0 && disc.empty() && !bdvd_dir.empty() && !m_title_id.empty()
+				&& resolved_path == GetCallbacks().resolve_path(vfs::get("/dev_bdvd/PS3_GAME/USRDIR/EBOOT.BIN"))
+				&& resolved_path != GetCallbacks().resolve_path(hdd0_boot) && fs::is_file(hdd0_boot)
+		   		&& psf::get_integer(psf::load(hdd0_game + m_title_id + "/PARAM.SFO").sfo, "BOOTABLE") == 1)
```

Comment:
The BOOTABLE flag is only set in HD games, not in game patches

Reply:
Hmm okay, odd that it was done this way, you had expect that there would a method to ensure EBOOT.BIN is bootable not just game data. I will add a category check then.

Reply:
It makes sense IMO, as you can't boot a patch on its own